cypress/support/step-definitions/button-toggle-group-steps.js

```diff
@@ -32,3 +32,7 @@ Then('input width is not set to {string}', (width) => {
 Then('label Align on preview is {string}', (direction) => {
   label().should($element => expect($element).to.have.css(TEXT_ALIGN, `${direction}`));
 });
+
+When('I click on Button Toggle Group {string}', (button) => {
```

Comment:
Could you change name of prop ? It would be more readable
```suggestion
When('I click on Button Toggle Group {string}', (buttonName) => {
```

Reply:
👍